### PR TITLE
[NAV-110] Modus Chip - add property closeIcon to display a different icon.

### PR DESCRIPTION
**Pull request by yohernandez**
## Description

Adding a property closeIcon to Modus Chip. The property has two values: remove, close. The default is remove. The new option is to display the icon-close following the guidelines https://modus.trimble.com/components/web/chips/styles/
Using the same padding-top and padding-bottom when the icons are visible.

References #1336

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Tested manually and in Storybook. Added a new e2e test to check if the close icon is displayed when the closeIcon property is set to 'close'.

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by msankaran0712 — CHANGES_REQUESTED** [review]
We probably should wait until we get a confirmation on the icon question in [issue ](https://github.com/trimble-oss/modus-web-components/issues/1336).

File: stencil-workspace/src/components/modus-chip/modus-chip.tsx [hunk 1]
```diff
@@ -109,9 +113,12 @@ export class ModusChip {
...
         ) : null}
         <span>{this.value}</span>
-        {this.showClose ? (
-          <IconRemove onClick={this.disabled ? null : (event) => this.onCloseClick(event)} size={'24'}></IconRemove>
-        ) : null}
+        {this.showClose ?
+          this.closeIcon == 'remove'? (
+            <IconRemove onClick={this.disabled ? null : (event) => this.onCloseClick(event)} size={'24'}></IconRemove>
+          ): (
+            <IconClose onClick={this.disabled ? null : (event) => this.onCloseClick(event)} size={'24'}></IconClose>
```
Comment (msankaran0712) [inline stencil-workspace/src/components/modus-chip/modus-chip.tsx:120]: for icon close, the size should be `18`. 

`24` is too big, 
image

**Review by msankaran0712 — CHANGES_REQUESTED** [review]

File: stencil-workspace/src/components/modus-chip/modus-chip.tsx (hunk 1 above)
Comment (yohernandez) [reply stencil-workspace/src/components/modus-chip/modus-chip.tsx]: updated the size. Thanks!

**Comment by msankaran0712** [comment]
@yohernandez Modus team confirmed that the rounded close button(existing) is correct and does not need a configuration to change it to the other icon shown in the style guide. 
We will be fixing the icon in the Modus style guide wherever required.
Please revert only those code changes.

**Comment by yohernandez** [comment]
> @yohernandez Modus team confirmed that the rounded close button(existing) is correct and does not need a configuration to change it to the other icon shown in the style guide. We will be fixing the icon in the Modus style guide wherever required. Please revert only those code changes.

Sure, thank you!

**Comment by yohernandez** [comment]
Closing this PR. Fixing the height in a different branch.

**Pull request closed without merging**